.ci/jenkins/Jenkinsfile.prod.nightly

```diff
@@ -301,13 +301,8 @@ pipeline {
                     'Please look here: ${BUILD_URL}', subject: 'OPENSHIFT SERVERLESS LOGIC ${PRODUCT_VERSION}:nightly-build #${BUILD_NUMBER}: ' + "${currentBuild.currentResult}", to: 'kie-jenkins-builds@redhat.com'
         }
         always {
-            archiveArtifacts artifacts: '**/*.maven.log', fingerprint: false, defaultExcludes: true, caseSensitive: true, allowEmptyArchive: true
-
-            echo 'Generating JUnit report...'
-            junit allowEmptyResults: true, healthScaleFactor: 1.0, testResults: '**/target/*-reports/TEST-*.xml'
```

Comment:
junit reports are not needed anymore or this caused that heap space problems?

Reply:
they are not needed because we dont run with tests. The heap space was in the `'**/*.maven.log` where it looks for those logs in every folder, and I believe because of big kie-tools repo it started to get the heap space.

But those files are actually in the workspace root, so I just took them directly from there.

Reply:
thnx for explanation.